ext/gd/gd.c

```diff
@@ -2738,6 +2783,16 @@ PHP_FUNCTION(imagegd2)
 }
 /* }}} */
 
+#ifdef HAVE_GD_BMP
+/* {{{ proto bool imagebmp(resource im [, string filename [, int compression]])
```

Comment:
I think $compression should be 'boolean' here (from reading the source of gdImageBmpCtx())